dcos/dcos-ui
# [COPS] DCOS-54937: Jobs list shows job twice if name matches group

Do not jobs twice if the folder and the job
start with the same name.

Closes https://jira.mesosphere.com/browse/DCOS-54937

## Testing
0. Go to jobs tab.
1. Create a job with id "frontend.renovate.foo".
2. Create a job with id "frontend.renovate-bot"
3. Open frontend folder.
4. Verify that you see a "renovate" folder and a "renovate-bot" job.
5. Open renovate folder.
6. Verify that you see just a "foo" job.
7. Verify that the added test makes sense.

## Trade-offs
See my comments.

## Dependencies
None.

## Screenshots
### Before
![Screenshot from 2019-08-06 14-40-15](https://user-images.githubusercontent.com/40791275/62537018-1fb37c00-b858-11e9-8584-1a96f8fc4fe5.png)

### After
![Screenshot from 2019-08-06 14-38-56](https://user-images.githubusercontent.com/40791275/62537023-22ae6c80-b858-11e9-8fc0-5a2b3814ede2.png)
